Formula/gpgme.rb

```diff
@@ -26,14 +26,9 @@ class Gpgme < Formula
   depends_on "libassuan"
   depends_on "libgpg-error"
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
-    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
-  end
-
   def install
     ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
+    ENV["SETUPTOOLS_USE_DISTUTILS"] = "stdlib"
```

Comment:
Do-not-merge for now since this may be better to handle inside `python@3.9`/`python@3.10` formulae.

Upstream `setuptools >= 60` change: https://github.com/pypa/setuptools/pull/2896

Reply:
This is the workaround used in other formulae, so I'm fine with this. We should add a comment documenting this, though.

I also wonder if these patches should be handling this:

https://github.com/Homebrew/homebrew-core/blob/debd7efdd3adf35b875926a80f1e69ea126e62c4/Formula/python%403.10.rb#L62-L75

CC @Bo98

Reply:
We can add a comment like `git-cola` with possible fix if `distutils` is no longer needed with https://dev.gnupg.org/D545